packages/viz/examples/index.html

```diff
@@ -23,38 +23,33 @@
 
   <body>
     <section id="map"></section>
-    <div id="tooltip" style="position: absolute; z-index: 1; pointer-events: none; background-color: #FFF; padding: 8px;"></div>
 
     <!-- Map libraries-->
     <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.js"></script>
-    <script src="https://unpkg.com/deck.gl@8.0.14/dist.min.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.1.0-alpha.6/dist.min.js"></script>
 
     <!-- CARTO Deck.gl's Tile Layer -->
     <script src="/dist/viz.umd.js"></script>
 
     <script>
       async function initialize() {
-        cartoToolkitViz.setDefaultCredentials()
+        carto.setDefaultCredentials({ username: 'jbotella' });
```

Comment:
let's try to use a generic username for examples. We can reuse 'public', like for the blog, or maybe create a new 'web-sdk' user